code/game/machinery/vending.dm

```diff
@@ -321,6 +328,10 @@
 		visible_message("<span class='info'>\The [usr] swipes \the [ID_container] through \the [src].</span>")
 	var/datum/money_account/customer_account = get_account(I.associated_account_number)
 	if (!customer_account)
+		//Allow BSTs to take stuff from vendors, for debugging and adminbus purposes
+		if (I.assignment == "Bluespace Technician")

```

Comment:
I don't feel that this is the best way to go about solving this issue anyways.
Why not just add a loaded charge card to the BST's bag?